src/main/java/com/extrahardmode/features/AntiGrinder.java

```diff
@@ -162,16 +162,21 @@ public boolean onEntityDeath(EntityDeathEvent event)
         final boolean inhibitMonsterGrindersEnabled = CFG.getBoolean(RootNode.INHIBIT_MONSTER_GRINDERS, world.getName());
 
         // FEATURE: monsters which take environmental damage or spawn from spawners don't drop loot and exp (monster grinder inhibitor)
-        if (inhibitMonsterGrindersEnabled && entity instanceof Monster && entity.getType() != EntityType.SQUID)
+        if (inhibitMonsterGrindersEnabled && entity instanceof Monster)
         {
             if (EntityHelper.isLootLess(entity))
             {
                 clearDrops(event);
                 return false;
-            } else
+            }
+            else
             {   //Evaluate if this kill was a too easy kill
                 switch (entity.getType())
                 {
+                    case WITHER:
```

Comment:
Squid is an animal - and thus is redundant (unless I'm wrong and they inherit Monster)